cfd_protocol/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 mod oracle;
 mod protocol;
 
+pub mod curve_handlers;
```

Comment:
I would like all of this code to move into the `daemon` crate, it doesn't really belong to the protocol as such. It is an implementation detail of our application which lives inside the `daemon`.

As an added benefit, defining the code within `daemon` should correctly flag any kind of unused code once it is integrated.

Reply:
Can do! Lemme get an API sorted and I'll move it all over.